storage_service/locations/views.py

```diff
@@ -46,6 +46,11 @@ def package_list(request):
     packages = Package.objects.all()
     return render(request, 'locations/package_list.html', locals())
 
+def package_fixity(request, uuid):
```

Comment:
Be wary of shadowing common libraries like `uuid`, since it can cause weird bugs later.